docs/04. theming guide/09. custom_twig_functions_and_filters.md

```diff
@@ -0,0 +1,24 @@
+# Curstom twig functions and filters
+
+## Functions
+
+### Backend
+@todo complete this
```

Comment:
Please complete documentation

Reply:
Don't have time for it now and it is not a blocking issue

Reply:
in the pr for this I added a comment that maybe someone can finish this for hacktoberfest

Reply:
removed the code for the filters #77d439f9c so this wouldn't block this feature